Development.md

```diff
@@ -0,0 +1,77 @@
+# Development guide
+
+This guide will help you set up the `generator` locally, run tests, and use Docker for isolated testing.
+
+## Getting started
```

Comment:
first person should fork the repo and setup the fork properly and configure it as described in https://github.com/asyncapi/community/blob/master/git-workflow.md